### PR TITLE
[DAT-684] - Show or hide payment method options depending of billing country

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Show or hide payment method options depending of billing country

- If the country is **Argentina**, **"Transfer"** and **"Mercado Pago"** must be **displayed**.
- If the country is **Mexico** or **Colombia**, **"Transfer"** must be **shown** and **"Mercado Pago"** **hidden**.
- Any **other country** must be **hidden** **"Transfer"** and **"Mercado Pago"**.

**Notes:** If the user has selected credit card as payment method  the if the transfer is showing, it should be "disabled"

**Task:** [DAT-684](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-684)

**Screenshots:**

- *Payment method:** Credit Card and **Billing Country:** 'AR'

**Before:**

image

**After:**

image

- *Payment method:** Credit Card and **Billing Country:** 'CL'

**Before:**

image

**After:**

image

- *Payment method:** Credit Card and **Billing Country:** 'MX'

**Before:**

image

**After:**

image

### CODE REVIEW

**Review by bamaro-dev — COMMENTED** [review]
✔️

File: src/components/ChangePlanDeprecated/Checkout/PaymentMethod/PaymentMethod.js [hunk 1]
```diff
@@ -70,14 +71,16 @@ const PaymentMethodField = ({ billingCountry, paymentMethodType, optionView, han
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
   const allowTransfer = countriesAvailableTransfer.filter((c) => c === billingCountry)[0];
+  const allowMercadoPago = countriesAvailableMercadoPago.filter((c) => c === billingCountry)[0];
```
Comment (jcamposmk) [inline src/components/ChangePlanDeprecated/Checkout/PaymentMethod/PaymentMethod.js:74]: why don't you use find instead of filter?

Comment (jhoffman-ms) [reply src/components/ChangePlanDeprecated/Checkout/PaymentMethod/PaymentMethod.js:74]: good, I will replace to use find instead of filter

Comment (jhoffman-ms) [reply src/components/ChangePlanDeprecated/Checkout/PaymentMethod/PaymentMethod.js]: Done

**Pull request merged**